gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
+<figure><img src="../.gitbook/assets/image (4) (1).png" alt=""><figcaption><p>Proof cost</p></figcaption></figure>
 
-Create Orders  section can be used to place the following orders
+* Proof generation time — a history of how long did proof generation take from the time
+the order was matched to the time a proof was submitted to the market.
 
-1. **Buy Order**: A buy order is posted by a proof requester. Each buy order comprises of the following parameters:
-   1. **Cost**: Amount the user is willing to pay for the proof. This is a number represented in USD.
-   2. **Order Time Out**: Once the order enters the marketplace, how long should it stay in the order book. If it times out before being matched, it is purged. This is a number representing   Minutes.
-   3. **Generation Time**:  Once the order is matched, this specifies the maximum time the proof generator has to publish the proof. This is an optional parameter. This is a number expressed in minutes.&#x20;
-   4. **Public Inputs:** This is an optional parameter to add any public inputs if the circuit requires it.
+<figure><img src="../.gitbook/assets/image (6) (1).png" alt=""><figcaption><p>Proof generation time</p></figcaption></figure>
 
-<figure><img src="../.gitbook/assets/image (3) (1).png" alt=""><figcaption><p>Buy Order</p></figcaption></figure>
+### Order book
 
+The order book shows the request and proposals on the marketplace for orders which have not yet matched.
 
+<figure><img src="../.gitbook/assets/image (4).png" alt=""><figcaption><p>Order book view</p></figcaption></figure>
 
-2\. **Sell Order**: A proof Generator posts a sell order. Each sell order comprises the following parameters
+You can toggle to see your orders in the book and group them.
 
-* **Cost:** Amount the proof generator is willing to accept to create a proof.
-* **Order Time Out**: Once the order enters the marketplace, how long should it stay in the order book. If it times out before being matched, it is purged. This is a number representing   Minutes.
-* **Generation Time:** The upper bound by which the proof will be generated once matched.
+### Create orders
 
-<figure><img src="../.gitbook/assets/image (5) (1).png" alt=""><figcaption><p>Sell Order</p></figcaption></figure>
+Create orders section can be used to place orders:
 
-### Manage Orders
+* **Buy order**: A buy order is posted by a proof requester.
+  Each buy order comprises the following parameters:
+  * **Cost** — the amount in USD that the user is willing to pay for the proof.
+  * **Order time out** — time in minutes; how long should the order stay in the order book
+    once it enters the marketplace.
+    If it times out before being matched, the order will be purged.
+  * **Generation time** — the maximum time the proof generator has to publish the proof once the order
+    is matched.
+    This is an optional parameter expressed in minutes.
+  * **Public inputs** — an optional parameter to add public inputs if the circuit requires it.
```

Comment:
Here we have a typical situation: there are input variables, and we should describe their 1) meaning and 2) data type.

It's best to do it consistently between all variables.
```suggestion
  * **Cost** (in USD): the price that the user is willing to pay for the proof.
  * **Order timeout** (in minutes): how long an order stays in the order book
    once it enters the marketplace.
    If it times out before being matched, the order will be purged.
  * **Generation time** (in minutes, optional): the maximum time the proof generator has to publish the proof
  once the order is matched.
  * **Public inputs** (optional): public input data if the circuit requires it.
```

Reply:
fixed

Reply:
> Maybe we should only describe order types on the Terminology page, link it from the Frontend
Done that